TLM/TLM/U/UIScaler.cs

```diff
@@ -5,11 +5,46 @@ namespace TrafficManager.U {
     using UnityEngine;
 
     public static class UIScaler {
+        public const float GUI_WIDTH = 1920f;
+        public const float GUI_HEIGHT = 1080f;
+
+        /// <summary>Caching because UIView.Instance.uiCamera can be null sometimes.</summary>
+        private static float cachedGuiWidth = GUI_WIDTH;
+
+        /// <summary>Caching because UIView.Instance.uiCamera can be null sometimes.</summary>
+        private static float cachedGuiHeight = GUI_HEIGHT;
+
         /// <summary>Screen width for GUI is always fixed at 1920.</summary>
-        public static float GuiWidth => Singleton<UIView>.instance.uiCamera.pixelWidth;
+        public static float GuiWidth {
+            // TODO: Double check if GUI never changes width, the code below can be a const
+            get {
+                UIView uiView = Singleton<UIView>.instance;
+                if (uiView != null) {
```

Comment:
Wait... that's always true

Reply:
Somehow either uiView or instance was null while rendering overlays, that's what the user reported in #838

Reply:
Ok, I've checked log and nullpointer comes from `Camera` which is `uiCamera` property of `UIView`. 
You've added null check below for that, which should solve problem with exception.

The real problem is why uiCamera was null? Game was already loaded so it seems you tried using old instance, probably destroyed. If you look closely at log you will notice that he loads game multiple times (with exit to main menu)

Reply:
One guess: many users reporting severe overlay rendering artifacts (overlapping with UI) seem to have overlays always enabled in the options. Which might render from the Tool code early before the rest of the UI has finished setting up? Just a guess. Could be true or not.

Reply:
What's my action on this? Can I safely assume instance is always non-null? I'd rather be on the safe side and later would possibly eliminate this class entirely. But now more worried to do minimal changes and solve the crash situation.

Reply:
Be over cautious for now and add an issue to remind more in-depth alterations later?

Reply:
Ok, now I see where is the problem. 
You shouldn't use `Singleton<UIView>` because that way you will create and add new `UIView` each time you call it (everytime with null camera). 
Use `UIView.GetAView()` to get correct `UIView` instance (the main one).
Call to `Singleton.instance` won't work because `UIView` class not extends `Singleton<>` but `MonoBehaviour` directly

Reply:
Wow! Maybe this is what's causing #822 too!